recipes/itango/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "itango" %}
+{% set version = "0.1.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 30346418242383484cde347abc93d81fde0147f2f49de71d3edf05a5eb73d098
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  entry_points:
+  # As the package is noarch, we can't use py2k / py3k selector
+  # Create both itango and itango3 entry points
+    - itango = itango:run
+    - itango3 = itango:run
+    - itango-qt = itango:run_qt
+    - itango3-qt = itango:run_qt
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
```suggestion
    - python >=3.6
```

Reply:
That's only correct if the package is `noarch: python` (see my comment above).

Reply:
I used `python >=2.7` to make it possible for someone to build the recipe with Python 2. Is it OK?